tests/test_dagmc_file.py

```diff
@@ -14,27 +13,9 @@ def test_load_file():
     """Tests loading file and check the values of all the variables that are
     set in the constructor
     """
-
-    single_cube = ds.DagmcStats(test_env[1]['input_file'])
+    single_cube = df.DagmcFile(test_env[1]['input_file'])
```

Comment:
We probably want the same changes that were maze in the query tests for this data structure.